src/components-examples/material/stepper/stepper-intl/stepper-intl-example.html

```diff
@@ -0,0 +1,59 @@
+<label>Pick the text for the optional label</label>
+<mat-radio-group
+  aria-label="Select an option for the optional label"
```

Comment:
I think rather than a `aria-label`, you probably just want to give this an id like `id="demo-optional-label-group"` and add `for="demo-optional-label-group"` to the `<label>` above